tools/apply-functions.py

```diff
@@ -0,0 +1,16 @@
+functions = {}
```

Comment:
Unless there is some documentation on how to use this script, I don't think we should commit it to the public repo.